src/main/java/com/vkatit/cinema/config/SpringConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.vkatit.cinema.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+@Configuration
+@ComponentScan("com.vkatit.cinema")
+@PropertySource("classpath:application.properties")
+public class SpringConfig {
+    private final Environment environment;
+
+    @Autowired
+    public SpringConfig(Environment environment) {
+        this.environment = environment;
+    }
+
+    @Bean
+    public Connection dataSource() {
+        Connection connection = null;
```

Comment:
better to use postgres driver programmaticaly. also all properties should be extracted using @Value annotation.